transform/mattermost-analytics/models/reports/product/mme/rpt_won_opportunities.sql

```diff
@@ -58,28 +60,47 @@ with account_hierarchy as (
     where license_id is not null
     qualify row_number() over (partition by server_id, license_id, license_telemetry_date order by timestamp desc) = 1
 ), all_telemetry_reported_licenses as (
-    select license_id, server_id from mm_telemetry_prod_license
+    select license_id, server_id, license_telemetry_date from mm_telemetry_prod_license
     union
-    select license_id, server_id from mattermost2_license
+    select license_id, server_id, license_telemetry_date from mattermost2_license
+), latest_active_users as (
+    select
+        server_id
+        , last_activity_date
+        , last_daily_active_users
+        , last_monthly_active_users
+        , last_count_registered_active_users
+    from
+        {{ ref('int_server_telemetry_summary') }}
+    where server_id in (select server_id from all_telemetry_reported_licenses)
 )
 select
-    opportunity_id
-    , account_id
-    , account_name
-    , root_account_id
-    , root_account_name
-    , account_type
-    , root_account_type
-    , account_arr
-    , root_account_arr
-    , is_latest
+    o.opportunity_id
+    , o.account_id
+    , o.account_name
+    , o.root_account_id
+    , o.root_account_name
+    , o.account_type
+    , o.root_account_type
+    , o.account_arr
+    , o.root_account_arr
+    , o.is_latest
+    , o.license_id
     , kl.sku_short_name as license_sku
     , kl.licensed_seats
     , kl.expire_at
     , l.license_id is not null as has_telemetry
-    , array_unique_agg(l.server_id) as servers
+    , min(datediff('day', l.license_telemetry_date, current_date)) as days_since_last_license_telemetry
+     , array_unique_agg(l.server_id) as servers
+    -- Consider active servers with MAU > 0
+    , array_unique_agg(st.server_id) as active_servers
+    , array_size(active_servers) > 0 as has_user_activity
+    , min(datediff('day', st.last_activity_date, current_date)) as days_since_last_activity
```

Comment:
@ifoukarakis - I think I missed this `days_since_last_activity`- > `days_since_last_user_activity`